Bonsai.Templates/Bonsai.PackageTemplate/Bonsai.PackageTemplate.csproj

```diff
@@ -47,9 +47,6 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.VisualStudio.CoreUtility">
-      <Private>False</Private>
-    </Reference>
```

Comment:
I don't even remember what this was anymore, I guess happy that it's gone.

Reply:
Yeah I couldn't find a straight answer on what this was either, but it was causing problems and didn't seem to be being used so off it went.